### PR TITLE
Log error of OnConfigurationChange hook.

### PR DESCRIPTION
**Pull request by andreiavrammsd**
#### Summary
If `OnConfigurationChange` returns an error at plugin install, the error can be seen in logs. But if there is an error at a later time, when configuration is changed, there is no trace of what happened.

### CODE REVIEW

**Comment by hanzei** [comment]
@lieut-data @levb Requesting your review on this PR given that you (if I recall correctly) had a discussion about what do to with the returned error. If you need additional, feel free to also request my review.

**Comment by hanzei** [comment]
@andreiavrammsd  Thank you very much for your PR. I've queued it for dev review, but given that the holidays are starting the review might not start until next year.

File: app/plugin.go [hunk 1]
```diff
@@ -197,7 +197,9 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 		a.SyncPluginsActiveState()
 		if pluginsEnvironment := a.GetPluginsEnvironment(); pluginsEnvironment != nil {
 			pluginsEnvironment.RunMultiPluginHook(func(hooks plugin.Hooks) bool {
-				hooks.OnConfigurationChange()
+				if err := hooks.OnConfigurationChange(); err != nil {
+					a.Log.Error(err.Error())
```
Comment (lieut-data) [inline app/plugin.go]: Can we add some context to the log?
```suggestion
					a.Log.Error("Plugin OnConfigurationChange hook failed", mlog.Err(err))
```

Comment (andreiavrammsd) [reply app/plugin.go]: I can't test this offline right now. I've commited your suggestion and we'll see what the automated tests say.

**Comment by hanzei** [comment]
@lieut-data This PR slipped through your review queue. Gentle reminder to review it.

**Comment by lieut-data** [comment]
@prapti, testing this should just require a plugin implementing `OnConfigurationChange` returning a non-nil `err`, then trying an arbitrary system console change, and verifying we get an error log each time.

**Comment by prapti** [comment]
Is there a Jira ticket associated with this PR?

**Comment by jasonblais** [comment]
@prapti As far as I know there is not. Would you be open to creating one?

**Comment by levb** [comment]
@prapti ping

**Comment by hanzei** [comment]
Thanks for the small but very impact full fix @andreiavrammsd :+1:

**Comment by andreiavrammsd** [comment]
Thank you for your time!

**Pull request merged**